### PR TITLE
Add Linked Events docs

### PR DESCRIPTION
**Pull request by rchinn-segment**
### Proposed changes

- Added Linked Events doc and setup guides for private beta
### Merge timing

- Target date is 9/20

### CODE REVIEW

File: src/connections/destinations/actions.md [hunk 1]
```diff
@@ -118,11 +118,13 @@ If necessary, click **New Mapping** to create a new, blank action.
 1. In the edit panel, define the [conditions](#conditions) under which the action should run.
 2. Test those conditions to make sure that they correctly match an expected event.
     This step looks for events that match the criteria in the [debugger queue](/docs/connections/sources/debugger/), so you might need to Trigger some events with the expected criteria to test your conditions. You can skip the test step if needed, and re-try it at any time.
-3. Next, set up the data mapping from the Segment format to the destination tool format.
-4. Test the mapping with data from a sample event.
+3. Select data models to [enrich your events](/docs/unify/linked-profiles/linked-events/) with.
+4. Next, set up the data mapping from the Segment format to the destination tool format.
```
Comment (stayseesong) [inline src/connections/destinations/actions.md:122]:
```suggestion
4. Set up the data mapping from the Segment format to the destination tool format.
```

File: src/unify/linked-profiles/setup-guides/redshift-setup.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Grant table permissions based on customer need. Learn more about Amazon's [table permissions](https://docs.aws.amazon.com/redshift/latest/dg/r_GRANT.html){:target="_blank"}.
+
+```ts
+-- query data from a specific table in a schema
+GRANT SELECT ON TABLE <schema-name>.<table-name> TO ROLE segment_entities;
+```
+
+### RETL table permissions
+
+If you've used RETL in your database, you'll need to add the following [table permissions](https://docs.aws.amazon.com/redshift/latest/dg/r_GRANT.html){:target="_blank"}:
```
Comment (stayseesong) [inline src/unify/linked-profiles/setup-guides/redshift-setup.md:57]:
```suggestion
If you used RETL in your database, you'll need to add the following [table permissions](https://docs.aws.amazon.com/redshift/latest/dg/r_GRANT.html){:target="_blank"}:
```

File: src/unify/linked-profiles/linked-events.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+- The edit panel shows you the mapping output in the format for the destination tool. You can change your mapping as needed and re-test.
+2. When you're satisfied with the mapping, click **Save**. Segment returns you to the Mappings table.
+
+> warning ""
+> At this time, when you select mappings or test events, you won’t see enrichment data. Enrichment data is only available with real events.
+
+## Frequently asked questions
+
+{% faq %}
+{% faqitem What data warehouse permissions does Segment require? %}
```
Comment (stayseesong) [inline src/unify/linked-profiles/linked-events.md:156]: how come it's being formatted like this? I think we stopped using these faq tags and just moved on to putting questions as h4 headers

File: src/unify/linked-profiles/linked-events.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+You can choose which database within your warehouse this data lives in. 
+
+> info ""
+> Linked Events syncs data from your warehouse approximately once every hour. 
+
+### Supported data warehouses
+
+The table below shows the data warehouses Linked Events supports. View the Segment docs for your warehouse, then carry out the corresponding steps. 
+
+| **Table**              |      **Steps**          |
```
Comment (stayseesong) [inline src/unify/linked-profiles/linked-events.md:79]:
```suggestion
|  Table              |      Steps          |
```

Comment (stayseesong) [reply src/unify/linked-profiles/linked-events.md:79]: Instead of Table, shouldn't it say Data Warehouse?

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/unify/linked-profiles/linked-events.md (hunk 3 above)
Comment (rchinn-segment) [reply src/unify/linked-profiles/linked-events.md:156]: Oh, good callout. If we've moved away from these tags, I'll update the formatting here to remain consistent.

**Pull request merged**